hal/network/lwip/ppp_client.cpp

```diff
@@ -168,6 +172,7 @@ int Client::input(const uint8_t* data, size_t size) {
       case STATE_CONNECTING:
       case STATE_DISCONNECTING:
       case STATE_CONNECTED: {
+        LOG(TRACE, "RX: %lu", size);
```

Comment:
Do we need this in non-debug builds? I would argue that this seems too verbose for every packet. Same in https://github.com/particle-iot/device-os/pull/1994/files#diff-a2527e86865833a37b99004691cf38a7R363

Reply:
These logs are already present in the form of the LWIP layer, but harder to enable by this method. I’d say there’s much less logs in the modular build, just the right amount. There was already a ton in monolithic so I don’t think one extra RX to match the TX that was already there will hurt. Plus I reduced the text from “Outputing XX bytes” to “TX: XX”

Reply:
These logs are already present in the form of the LWIP layer, but harder to enable by this method. I’d say there’s much less logs in the modular build, just the right amount. There was already a ton in monolithic so I don’t think one extra RX to match the TX that was already there will hurt. Plus I reduced the text from “Outputing XX bytes” to “TX: XX”

Reply:
Also I would say we do need this for minimal debugging purposes and it can be easily filtered away.